package.json

```diff
@@ -23,7 +23,7 @@
   },
   "dependencies": {
     "@reduxjs/toolkit": "^1.8.1",
-    "@yext/answers-core": "^1.6.0-beta.7",
+    "@yext/answers-core": "^1.7.0-alpha.165",
```

Comment:
do we also need to update the headless version to publish a new version? I wasn't sure if we were trying to export these endpoints from headless-react or the component library too

Reply:
I forgot to bump the version! updated now